apps/i18n/common/en_us.json

```diff
@@ -1866,6 +1866,29 @@
   "myProjects": "My Projects",
   "myProfessionalLearningCourses": "My Professional Learning Courses",
   "myWorkshops": "My Workshops",
+
+  "accountInformationAccountInformation": "Account information",
+  "accountInformationVerifiedTeacher": "Verified Teacher",
+  "accountInformationDisplayName": "Display Name",
+  "accountInformationUsername": "Username",
+  "accountInformationDisplayNameHintTeacher": "This is what your students see.",
+  "accountInformationEmail": "Email address",
+  "accountInformationEmailHint": "Make sure you can receive emails to this email address.",
+  "accountInformationPassword": "Password",
```

Comment:
The string password already existed in this file: https://github.com/code-dot-org/code-dot-org/blob/fcc0ce8fbcc11d556b3aee3fe2c7cc2d18b81f63/apps/i18n/common/en_us.json#L2050

Unless we want a translation that requires more context, where translations may change based on that context, we prefer to reuse generic strings. It's always a good practice to check whether the string I'm trying to add already exists or not.